guides/plugins/apps/content/cms/add-custom-cms-blocks.md

```diff
@@ -217,9 +217,12 @@ In `Resources/views/storefront/block/` a Twig template matching the pattern `cms
 
 So in this example, it's sufficient to simply extend the existing `image-text` element:
 {% code title="<app root>/Resources/views/storefront/block/cms-block-swag-image-text-reversed-component.html.twig" %}
+{% raw %}
 ```text
 {% sw_extends '@Storefront/storefront/block/cms-block-image-text.html.twig' %}
 ```
+{% endraw %}
+
```

Comment:
Is this new line needed?

Reply:
No, it's It's an artifact of the fix I'll get rid of it :+1: